index.rst

```diff
@@ -1,7 +1,7 @@
 Citus Documentation
 ====================
 
-Welcome to the documentation for Citus 6.0! Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.
+Welcome to the documentation for Citus 6.1! Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.
```

Comment:
If you want to bump the overall docs version in preparation for the new version tag then you'll probably want to update `conf.py` as well.